admin_guide/topics/create_aggregated_cluster_role_crd.adoc

```diff
@@ -0,0 +1,78 @@
+////
+create aggregated cluster role definitions for a custom resource definition
+
+Module included in the following assemblies:
+
+* admin_guide/custom_resource_definitions.adoc
+////
+
+[id='create-aggregated-cluster-role-crd-{context}']
+= Creating cluster roles for the custom resource definition
+
+After you create cluster-scoped CRDs, you can grant permissions to it. If you
+use the admin, edit, and view 
+xref:../../architecture/additional_concepts/authorization.adoc#roles[default
+cluster roles], take advantage of cluster role aggregation for their rules.
+
+[IMPORTANT]
+====
+You must explicitly assign permissions to each of these roles. The roles with
+more permissions do not inherit rules from roles with fewer permissions. If you
+assign a rule to a role, you must also assign that verb to roles that have more
+permissions. For example, if you grant the "get crontabs" permission to the view
+role, you must also grant it to the edit and admin roles. The admin or edit role 
+is usually assigned to the user that created a project through the project 
+template.
```

Comment:
link to https://docs.openshift.org/latest/admin_guide/managing_projects.html#template-for-new-projects